assets/_js/custom-viz/boeing/waypoints.js

```diff
@@ -85,25 +89,50 @@ const Waypoints = () => {
     })
   })
 
-  //if paragraph hits top of screen, part label disappears
   paragraphs.forEach(paragraph => {
-    const allParts = Array.from(document.querySelectorAll('.part'))
+    // const allParts = Array.from(document.querySelectorAll('.part'))
     const activeParts = Array.from(paragraph.querySelectorAll('.part'))
+    // const activeSVGs = []
 
     new Waypoint({
       element: paragraph,
       handler: function() {
-        allParts.forEach(function(part) {
+        const previousWaypoint = this.previous()
```

Comment:
I would move this variable declaration so it's above the section of code that it's relevant to.